dataclass_function.py

```diff
@@ -1,41 +1,43 @@
-import datetime
+# import datetime
+# from datetime import date
 from dataclasses import dataclass
 
+import pandas as pd
 from pandas_dataclasses import AsFrame, Data, Index
 
 
 @dataclass
 class Pupil(AsFrame):
     """Pupil information"""
 
-    pupilno: Index[str]
-    rreduserID: Data[str]
-    assessi_engtest2: Data[int]
-    assessi_iretest1: Data[int]
-    assessi_iretype1: Data[int]
-    assessi_maltest1: Data[int]
-    assessi_outcome: Data[int]
-    assessi_scotest1: Data[int]
-    assessi_scotest2: Data[int]
-    assessi_scotest3: Data[int]
-    assessii_engcheck1: Data[int]
-    assessii_engtest4: Data[int]
-    assessii_engtest5: Data[int]
-    assessii_engtest6: Data[int]
-    assessii_engtest7: Data[int]
-    assessii_engtest8: Data[int]
-    assessii_scotest4: Data[int]
-    assessii_iretest2: Data[int]
-    assessii_iretype2: Data[int]
-    assessiii_engtest10: Data[int]
-    assessiii_engtest11: Data[int]
-    assessiii_engtest9: Data[int]
-    assessiii_iretest4: Data[int]
-    entry_dob: Data[datetime.datetime]
+    pupil_no: Index[str]
+    rred_user_id: Data[str]
+    assessi_engtest2: Data[float]

```

Comment:
out of interest does this work from pandas? Would be preferable if it did
```suggestion
    assessi_engtest2: Data[Int32Dtype]
```


 If not then should add a comment the first time so that we know why we're using floats

Reply:
This doesn't work :( so will just add a comment on why